docs/pages/_layouts/gh-basic.html

```diff
@@ -91,9 +91,12 @@
     <span>Reference</span>
```

Comment:
What does this change?

Reply:
This section is creating the docs page menu. This change updates it to automatically populate entries against whatever's in the `reference/` directory, mirroring the `tutorials/` behavior a few lines up. At the moment the only thing in `reference/` is `yaml-reference`.